cwl/src/test/scala/wdl4s/cwl/CwlWorkflowWomSpec.scala

```diff
@@ -124,10 +184,8 @@ class CwlWorkflowWomSpec extends FlatSpec with Matchers {
     val wc = nodes.collectFirst({ case wc: CallNode if wc.name == "wc" => wc }).get
 
     ps.upstream shouldBe empty
-    cgrep.upstream.filter(_ eq ps).size shouldBe 1
-    cgrep.upstream.filter(_ eq cgrepPatternInput).size shouldBe 1
-
-    wc.upstream.filter(_ eq ps).size shouldBe 1
+    cgrep.upstream should contain theSameElementsAs Set(ps, cgrepPatternInput)
```

Comment:
does `theSameElementsAs` do reference equality?  Important as that assertion broke this test earlier as we were creating duplicate equivalent instances.